src/toNumber.js

```diff
@@ -0,0 +1,22 @@
+import { ifElse, always } from 'ramda';
+
+import isCoercible from './internal/isCoercible';
+
+/**
+ * Converts value to a number.
+ *
+ * @func toNumber
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.29.0|v2.29.0}
+ * @param {*} val The value to convert
+ * @return {Number}
+ * @example
+ *
+ * toNumber('1'); //=> 1
```

Comment:
I'm suggesting following examples here. The examples derives from [lodash.toNumber examples](https://lodash.com/docs/4.17.15#toNumber)

````js
RA.toNumber(3.2); //=> 3.2
RA.toNumber(Number.MIN_VALUE); //=> 5e-324
RA.toNumber(Infinity); //=> Infinity
RA.toNumber('3.2'); //=> 3.2
RA.toNumber(Symbol('3.2')); //=> NaN